src/Symfony/Component/ObjectMapper/Metadata/Mapping.php

```diff
@@ -11,26 +11,15 @@
 
 namespace Symfony\Component\ObjectMapper\Metadata;
 
+use Symfony\Component\ObjectMapper\Attribute\Map;
+
 /**
  * Configures a class or a property to map to.
  *
  * @internal
  *
  * @author Antoine Bluchet <soyuka@gmail.com>
  */
-readonly class Mapping
+final readonly class Mapping extends Map
```

Comment:
Is this valid code ? I don't think a readonly class can extend a non-readonly one.